guides/source/7_1_release_notes.md

```diff
@@ -88,7 +88,37 @@ User.normalize_value_for(:phone, "+1 (555) 867-5309") # => "5558675309"
 
 ### Add `ActiveRecord::Base.generates_token_for`
 
-TODO: Add description https://github.com/rails/rails/pull/44189
+A new [method `generates_token_for`](https://github.com/rails/rails/pull/44189) has been introduce
+to `ActiveRecord::Base`. This feature allows to generate tokens that can embed data from a record.
```

Comment:
add 'one' between 'allows' and 'to'

Reply:
I think we can use 'you' instead on one.